src/frontend/js/widgets/Dashboard/utils/teacherRouteMessages.tsx

```diff
@@ -13,11 +13,12 @@ export enum TeacherDashboardPaths {
   ORGANIZATION_COURSES = '/teacher/organization/{organizationId}/courses',
   ORGANIZATION_SETTINGS = '/teacher/organization/{organizationId}/settings',
   ORGANIZATION_MEMBERS = '/teacher/organization/{organizationId}/members',
-  COURSE = '/teacher/course/{courseCode}',
-  COURSE_SETTINGS = '/teacher/course/{courseCode}/settings',
-  COURSE_CLASSROOMS = '/teacher/course/{courseCode}/classrooms',
-  COURSE_STUDENTS = '/teacher/course/{courseCode}/students',
-  COURSE_RECORDS = '/teacher/course/{courseCode}/records',
+  COURSE = '/teacher/course/{courseId}',
+  COURSE_PRODUCT = '/teacher/course/{courseId}/product/{courseProductRelationId}',
+  COURSE_SETTINGS = '/teacher/course/{courseId}/settings',
+  COURSE_CLASSROOMS = '/teacher/course/{courseId}/classrooms',
+  COURSE_STUDENTS = '/teacher/course/{courseId}/students',
+  COURSE_RECORDS = '/teacher/course/{courseId}/records',
```

Comment:
Let's ignore those routes for now.

Reply:
I thought about having a PR to remove unused route. Not to mix too may things in the same PR.

Reply:
Yup right !